specification/k3po.control/SPEC.md

```diff
@@ -192,7 +174,7 @@ N/A
 ##### NOTIFIED event
 
 ###### Description
-This event is sent by the driver when a barrier is triggered.
+This event notifys the client that a barrier has been triggered.  It may be sent after a barrier is triggered, or after a NOTIFY command if the barrier was previously completed.
 
```

Comment:
The NOTIFY command triggers the barrier. It does not matter if it was completed or not.